atd-vze/src/views/Tools/UnitsForm.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  Col,
+  Input,
+  Label,
+  FormGroup,
+  FormText,
+} from "reactstrap";
+import { useQuery } from "@apollo/react-hooks";
+import { withApollo } from "react-apollo";
+import { gql } from "apollo-boost";
+
+const GET_VEHICLE_DESC_LKP = gql`
+  {
+    atd_txdot__veh_unit_desc_lkp {
+      veh_unit_desc_id
+      veh_unit_desc_desc
+    }
+  }
+`;
+
+const UnitsForm = ({ units, handleUnitFormChange, client }) => {
+  const { data: lookupValues, loading } = useQuery(GET_VEHICLE_DESC_LKP);
+
+  return (
+    <>
+      {units.map((unit, i) => {
+        return (
+          <Card key={`unitCard-${i + 1}`}>
+            <CardHeader>Unit #{i + 1}</CardHeader>
+            <CardBody>
+              <FormGroup row>
+                <Col md="3">
+                  <Label htmlFor="unit-type">Unit Type</Label>
+                </Col>
+                <Col xs="12" md="9">
+                  <Input
+                    type="select"
+                    id="unit-type"
+                    name="unit-type"
+                    placeholder="Enter Fatality Count..."
+                    value={unit.unit_desc_id}
+                    onChange={e =>
+                      handleUnitFormChange({
+                        type: "unit_desc_id",
```

Comment:
Took me a second to realize this was a default case in the `unitsReducer`. Works great but it might be clearer to add another action type called `updateUnitField` or something. I guess the payload would have to be an object like `{"unit_desc_id": e.target.value }`. Hmmm.

It could also just be me digging too deeply into the cobwebs of Redux patterns in my head.  🙃